modules/test/playwright/tests/osb-faro-web/individuals.spec.ts

```diff
@@ -79,7 +79,8 @@ test(
 		});
 
 		await test.step('Go to Analytics Cloud and Switch the property', async () => {
-			await navigateToACSitesPageViaURL({
+			await navigateToACPageViaURL({
+				acPage: ACPage.sitePage,
```

Comment:
Hey @marcosapmf, I think there should be a navigation to the Individuals page and you could remove line 91:
`await navigateTo({page, pageName: 'Individuals'});`

```suggestion
			await navigateToACPageViaURL({
				acPage: ACPage.individualPage,
```